src/components/common/bullets-colin-link.vue

```diff
@@ -155,4 +170,9 @@ export default class BulletsColinLink extends Mixins(CommonMixin, NrAffiliationM
   margin-top: 1rem;
   padding-left: 8rem;
 }
+
+.nr-input {
+  padding-right: 1.5rem;
+  padding-left: 1.5rem;
+}
```

Comment:
You could also just assign the appropriate Vuetify helper class (px-6) on line 21.

Ref: https://v2.vuetifyjs.com/en/styles/spacing/

Reply:
removed this css and used px-6

Reply:
Fixed.